embedded-memsql/README.adoc

```diff
@@ -33,5 +34,6 @@
 * Images without "minimal" tag do no start within 30 secs, so they are unusable
```

Comment:
Are these notes still actual?
pls replace `Network latency is currently do not work` with `NetworkTestOperations are not supported currently`. 
Also what is the reason it does not work anymore?

Reply:
Imposible to install ifconfig via yum into docker image

Reply:
have you checked https://www.unixmen.com/ifconfig-command-found-centos-7-minimal-installation-quick-tip-fix/ ?